### ISSUE TITLE
Fullpage section not showing custom component

### ISSUE THREAD

**Issue by ulisses-cruz** [alvarotrigo/vue-fullpage.js#138]
### Description
Fullpage section not showing custom component

### Link to isolated reproduction with no external CSS / JS
https://codesandbox.io/embed/codesandbox-nuxt-svz7f

**Comment by alvarotrigo** [comment]
@vaso2 @dobromir-hristov @Chiaparini any insights ?

**Comment by ChromaticMind** [comment]
I have the same issue and I have a paid licence as well, please can someone get back to me on this ASAP?

`<template>
  <div>
    <no-ssr>
      <full-page id="fullpage" :options="options" rel="fullpage">
        <div class="section">
          <component :is="modules.CopyOnlyHero" />
        </div>
        <div class="section">
          <component :is="modules.FeaturedContent" v />
        </div>
      </full-page>
    </no-ssr>
  </div>
</template>`

The sections are blank and doing inspect shows:
`<div id="fullpage" rel="fullpage" class="fullpage-wrapper" style="height: 100%; position: relative; transition: none 0s ease 0s;"
<div class="section fp-section fp-table" data-fp-styles="null" style="height: 969px;">
<div class="fp-tableCell" style="height: 969px;"></div>
</div>
....`

**Comment by ChromaticMind** [comment]
Hey @vaso2  @dobromir-hristov  @Chiaparini  @alvarotrigo  any help here would be greatly appreciated.

@ulisses-cruz did you end up finding a solution?

**Comment by dobromir-hristov** [comment]
I will check it out later today.

**Comment by dobromir-hristov** [comment]
Still working at my day job 😆 Priorities mate hhehe

**Comment by dobromir-hristov** [comment]
OK so, here it goes.

Full-page extracts all of it's children on `mounted` hook, and builds the whole markup and stuff.
However, `mounted` hook of a parent component does not guarantee all children are mounted. As such, dynamically loaded components may render later, this means that the parent will not wait for them.

In our case, FullPage is just rendering what it has, with those async components getting trashed.

Unfortunately there is no way to wait for all child components to be ready out of the box.

## Proposed solution
We could add a prop to stop fullpage from initializing on mounted and instead do this manually. That way users can figure out when their components are mounted and init.

You could also call `this.$refs.fullpage.build()` when your components are ready, but that will just re-init the whole thing, causing unnecessary renders and flickers.

**Comment by ChromaticMind** [comment]
hey @dobromir-hristov, while we wait for that PR to merge in I thought I'd try `this.$refs.fullpage.build()`

I did this:

```
mounted() {
    setTimeout(() => {
      this.$refs.fullpage.build();
    }, 2000);
  },
```

But it does nothing. Still my sections are empty :(

**Comment by dobromir-hristov** [comment]
Yeah, that wont work. The dom has been augmented already :/ You could try my branch, but you need to pull it locally and do a `yarn link` to it.

**Comment by ChromaticMind** [comment]
I think Alvaro said to merge your PR into dev, perhaps once that is done I can pull that down?

**Comment by alvarotrigo** [comment]
Fixed on dev!

**Comment by ChromaticMind** [comment]
@alvarotrigo @dobromir-hristov thanks, I can confirm the issue of blank sections is no longer present when using `:skip-init="true"`.

**Comment by alvarotrigo** [comment]
Let's keep it open until I merge the dev branch into the master and create a new release.

**Comment by alvarotrigo** [comment]
This was fixed in 0.1.5
